### DISCUSSION TITLE
doris 1.2.1 Maximum number of arguments for aggregate function with Nullable types is 8

### DISCUSSION THREAD

**Discussion by listen2012** [apache/doris#15901] [A - General / Q&A]
when the number of arguments for some agg functions (sequence_match, retention..) more than 8 ，be will crash down.

部分agg函数输入参数数量不能超过8个，否则be宕机

sql：
SELECT 
    uid,     
    retention(date = '2022-10-12',date = '2022-10-13',date = '2022-10-14',date = '2022-10-15'
    ,date = '2022-10-16',date = '2022-10-17',date = '2022-10-18',date = '2022-10-19'
    ,date = '2022-10-20')
    AS r 
    FROM sequence_match_test1 
    GROUP BY uid 
    ORDER BY uid ASC;

message：F0113 15:01:20.535117 11500 aggregate_function_null.h:279] Maximum number of arguments for aggregate function with Nullable types is 8

**Comment by Yukang-Lian** [comment]
At present, we only support a maximum of 8 parameters. If there is a need, we will discuss to increase the maximum number of supported parameters.

**Comment by Yukang-Lian** [comment]
Hi @listen2012 , we will remove the limit on the max number of parameters, and it will be released in version 1.2.2. Thanks for your support!

**Reply by listen2012** [reply]
ok，thanks for the reply

**Reply by listen2012** [reply]
hello, when the version 1.2.2 will be released?

**Reply by Yukang-Lian** [reply]
February.
